backend/src/main/java/ai/verta/modeldb/authservice/PublicRoleServiceUtils.java

```diff
@@ -266,6 +268,18 @@ public WorkspaceDTO getWorkspaceDTOByWorkspaceId(
     return Collections.emptyList();
   }
 
+  @Override
+  public boolean createWorkspaceRoleBinding(
```

Comment:
Note: this will create more than just workspace role bindings. It will set owner and visibility too.

Reply:
Easy to rename it now